examples/identity/identity.bal

```diff
@@ -6,16 +8,22 @@ public function main() {
     MyClass obj1 = new MyClass();
     MyClass obj2 = new MyClass();
     // `b1` will be true.
-    boolean _ = (obj1 === obj1);
+    boolean b1 = (obj1 === obj1);
     // `b2` will be false.
-    boolean _ = (obj1 === obj2);
+    boolean b2 = (obj1 === obj2);
     // `b3` will be true.
-    boolean _ = ([1,2,3] == [1,2,3]);
+    boolean b3 = ([1, 2, 3] == [1, 2, 3]);
     // `b4` will be false.
-    boolean _ = ([1,2,3] === [1,2,3]);
+    boolean b4 = ([1, 2, 3] === [1, 2, 3]);
     // `b5` will be true.
-    boolean _ = (-0.0 == +0.0);
+    boolean b5 = (-0.0 == +0.0);
     // `b6` will be false.
-    boolean _ = (-0.0 === +0.0);
+    boolean b6 = (-0.0 === +0.0);
 
+    io:println(b1);
+    io:println(b2);
+    io:println(b3);
+    io:println(b4);
+    io:println(b5);
+    io:println(b6);
```

Comment:
Shall we print these immediately after defining the variable? Easier to follow IMO.